tests/components/zha/zha_devices_list.py

```diff
@@ -2659,7 +2664,12 @@
                 "channels": ["power"],
                 "entity_class": "Battery",
                 "entity_id": "sensor.philips_rwl020_77665544_power",
-            }
+            },
+            ("binary_sensor", "00:11:22:33:44:55:66:77-2"): {
```

Comment:
```suggestion
            ("binary_sensor", "00:11:22:33:44:55:66:77-2-15"): {
```
After changing the registries, this should be a cluster specific match, not just endpoint